src/Sarif/Visitors/SarifTransformerUtilities.cs

```diff
@@ -13,6 +15,14 @@ namespace Microsoft.CodeAnalysis.Sarif.Visitors
 {
     public static class SarifTransformerUtilities
     {
+        private static readonly Regex CommitShaRegex = new Regex(@"^(?<hash>[0-9a-f]{40}).*$");
```

Comment:
>); [](http://example.com/codeflow?start=93&length=2)

Add , RegexOptions.Compiled) to all regexes

Reply:
Done. This is a good suggestion, thanks!